receiver/mongodbatlasreceiver/internal/mongodb_atlas_client.go

```diff
@@ -627,24 +627,15 @@ func (s *MongoDBAtlasClient) GetClusters(ctx context.Context, groupID string) ([
 }
 
 // GetAlerts returns the alerts specified for the set projects
-func (s *MongoDBAtlasClient) GetAlerts(ctx context.Context, groupID string, maxAlerts int64) ([]mongodbatlas.Alert, error) {
-	pageNum := 1
-	alertsResult := []mongodbatlas.Alert{}
-	for {
-		lo := mongodbatlas.ListOptions{PageNum: pageNum}
-		options := mongodbatlas.AlertsListOptions{ListOptions: lo}
-		alerts, response, err := s.client.Alerts.List(ctx, groupID, &options)
-		err = checkMongoDBClientErr(err, response)
-		if err != nil {
-			return nil, err
-		}
-		alertsResult = append(alertsResult, alerts.Results...)
-		if !hasNext(alerts.Links) || len(alertsResult) > int(maxAlerts) {
-			break
-		}
-		pageNum++
```

Comment:
I may have missed it, but it doesn't appear that we're bumping this anyways. Either way, this could be verified with a test.